api-docs/docs/browser-tracker/browser-tracker.api.md

```diff
@@ -72,13 +72,14 @@ export interface BrowserTracker {
     enableActivityTrackingCallback: (configuration: ActivityTrackingConfiguration & ActivityTrackingConfigurationCallback) => void;
     enableAnonymousTracking: (configuration?: EnableAnonymousTrackingConfiguration) => void;
     flushBuffer: (configuration?: FlushBufferConfiguration) => void;
-    getCookieName: (basename: string) => void;
-    getDomainSessionIndex: () => void;
-    getDomainUserId: () => void;
-    getDomainUserInfo: () => void;
+    getCookieName: (basename: string) => string;
+    getDomainSessionIndex: () => number;
+    getDomainUserId: () => string;
+    // Warning: (ae-forgotten-export) The symbol "ParsedIdCookie" needs to be exported by the entry point index.module.d.ts
```

Comment:
Shouldn't we export the `ParsedIdCookie` as part of this PR?

Reply:
Good call! I'd left it because that warning was already there for other unexported types, and it involved a change to the tracker itself rather than just the `-core` library. I have instances where the type is useful though, so have exported it from both.